bin/configs/rust-reqwest-petstore-async-tokensource.yaml

```diff
@@ -0,0 +1,11 @@
+generatorName: rust
+outputDir: samples/client/petstore/rust/reqwest/petstore-async-tokensource
```

Comment:
please add the new folder to the github workflow https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-rust.yaml so that CI will test it moving forward?

Reply:
Just want to confirm what we are talking about.

Do you want me to include the below mention path in the `push` section as well ? Currently the CI should automatically test it on pull requests right ?
![image](https://github.com/user-attachments/assets/366bc372-4acb-4067-acdb-f6d0f15c4b42)

Reply:
@nicolas-vivot I think you're right as it's tested as part of the workflow.